arches/app/utils/index_database.py

```diff
@@ -41,7 +41,10 @@ def get_serialized_graph(graph):
 
     if graph.graphid not in serialized_graphs:
         published_graph = graph.get_published_graph()
-        serialized_graphs[graph.graphid] = published_graph.serialized_graph
+
+        if published_graph:
+            serialized_graphs[graph.graphid] = published_graph.serialized_graph
+            
```

Comment:
If published_graph is false it looks like you're going to get a key error when the function tries to return `serialized_graphs[graph.graphid]`